src/web/pipeline/server.nginx.conf

```
@@ -62,14 +62,14 @@ server {
 			proxy_pass http://auth/user/;
 		}
 
-		location ~ ^/api/(space|schemes)/(.*)$ {
+		location ~ ^/api/(space|schemes)(/.*)? {
```

Доллар потерялся. (( Тут он конечно роли не играет, но если нам вдруг захочется что-то добавить в эту регулярку, мы можем забыть о нём и это может стать потенциальной дырой.
Давай писать ^ и $ во всех регулярках конфига nginx.
```suggestion
		location ~ ^/api/(space|schemes)(/.*)?$ {
```